airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -115,7 +115,7 @@ class S3Hook(AwsBaseHook):
     """
 
     conn_type = 's3'
-    hook_name = 'Amazon S3'
+    hook_name = 'Amazon S3 (Deprecated)'
```

Comment:
are you deprecating the _hook_ or the _conn type_?

here it looks like you're deprecating the hook?

Reply:
`hook_name` only use in UI and CLI (`airflow providers hooks`) for custom name of Connection as it described in base hook module 

https://github.com/apache/airflow/blob/5c52bbf32d81291b57d051ccbd1a2479ff706efc/airflow/hooks/base.py#L108-L111

![image](https://user-images.githubusercontent.com/3998685/186997694-35a1b702-cb78-41e8-9cbd-05552585da40.png)

Reply:
I do not know the better way to mark connection as deprecated in the UI.

So it is additional way for notify users that better to use Amazon Web Services instead of Amazon S3. In additional to documentation (simple page) and warning in task logs.

Reply:
ok sure, yeah it's weird that it's called hook  name but it really means conn name... but it's not the only weird thing :)

Reply:
do you not need to change `conn_type` as part of this deprecation?  forgive me for not recalling all the details of how this works...

Reply:
I don't think so. Just want to keep same behaviour in the ui so if user already create connection in the UI before then conn_type would `s3`.

But if change also conn_type I don't think old connection would associate with S3Hook and can't test connection in UI/API.

The main idea is remove `conn_type` and `hook_name` in the future release and after that this Connection Type removed from selected list in UI.